bigbluebutton-html5/imports/ui/components/breakout-room/component.jsx

```diff
@@ -210,25 +224,41 @@ class BreakoutRoom extends PureComponent {
     } = this.state;
 
     const roomItems = breakoutRooms.map(breakout => (
-      <div className={styles.content} key={`breakoutRoomList-${breakout.breakoutId}`}>
-        <span aria-hidden>
-          {intl.formatMessage(intlMessages.breakoutRoom, breakout.sequence.toString())}
-          <span className={styles.usersAssignedNumberLabel}>
-            (
-            {getNumUsersByBreakoutId(breakout.breakoutId)}
-            )
-          </span>
-        </span>
-        {waiting && requestedBreakoutId === breakout.breakoutId ? (
-          <span>
-            {intl.formatMessage(intlMessages.generatingURL)}
-            <span className={styles.connectingAnimation} />
+      <div className={styles.breakoutItems}>
```

Comment:
This div needs a key property. ``key={`breakoutRoomItems-${breakout.breakoutId}`}``